### DISCUSSION TITLE
Request Edit Access -  Google Docs Capstone Project

### DISCUSSION THREAD

**Discussion by FarhadFamser99** [teamalgoritma/community#911] [Q&A] [answered]
Pagi kak,
aku ga bisa akses edit untuk google docs capstone project guide
https://bit.ly/zno-capstone-da-guide
tapi tadi aku udah request access juga si dengan email a.n. farhadfamser02@gmail.com

mohon bantuannya ya kak 
terima kasih.

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @FarhadFamser99 

Terima kasih telah bertanya melalui Github Discussion Algoritma. 
Google Docs capstone project guide memang tidak bisa diakses lagi, pak, setelah hari pengerjaan kedua capstone (Jumat, 30 Agustus 2024). Akan tetapi, jika Bapak memiliki pertanyaan atau kendala terkait pengerjaan capstone, bisa ditanyakan melalui Github Discussion Algoritma ini, pak.

Sekian yang dapat kami sampaikan,

Best regards,
Team Algoritma

**Reply by FarhadFamser99** [reply]
baik kak terima kasih
